back-end/src/sciler/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+	"github.com/sirupsen/logrus"
+	"sciler/communication"
+	"sciler/config"
+	"time"
+)
+
+const host string = "localhost"
+const port string = "1883"
+
+var topics = []string{"back-end", "status", "hint", "connect"}
+
+// Message is a type that follows the structure all messages have, described in resources/message_manual.md
+type Message struct {
+	DeviceID string                 `json:"device_id"`
+	TimeSent string                 `json:"time_sent"`
+	Type     string                 `json:"type"`
+	Contents map[string]interface{} `json:"contents"`
+}
+
+func main() {
+	fmt.Println("Starting server")
+	data := []byte(`{
+            "name": "My Awesome Escape",
+            "duration": "00:30:00"
+    }`)
+	fmt.Println(config.GetFromJSON(data))
+
+	communicator := communication.NewCommunicator(host, port, topics)
+	go communicator.Start(func(client mqtt.Client, message mqtt.Message) {
+		// Todo: Make advanced message handler which acts according to the events / configuration
```

Comment:
should this todo be done for this issue? else make a new issue

Reply:
there is an issue for it already GH-21